jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcMetadataReader.java

```diff
@@ -236,7 +236,10 @@ private static String findSchema(String schema, DatabaseMetaData dbMeta, SqlDial
 
     private static List<TableMetadata> findTables(String catalog, String schema, List<String> tableFilter, DatabaseMetaData dbMeta, SqlDialect dialect) throws SQLException {
         List<TableMetadata> tables = new ArrayList<>();
-        ResultSet resTables = dbMeta.getTables(catalog, schema, null, null);
+        
+        String[] supportedTableTypes = {"TABLE", "VIEW", "SYSTEM TABLE"};
```

Comment:
Let's hope that the other data sources also follow the typical examples shown in the documentation (https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html#getTables(java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String[])) and do not filter out more, but I think the risk is low enough.